app/views/communicart_mailer/_comments.html.erb

```diff
@@ -0,0 +1,23 @@
+<% if comments.any? %>
+  <strong>Recent Comments</strong>
+  <table cellpadding='10'>
+    <% comments.each do |c| %>
+    <tr>
+      <td valign='top'>
+        <p>
+          <%= l(c.created_at.to_date) %>
+        </p>
+      </td>
+
+      <td valign='top'>
+        <p>
+          <% unless c.user.nil? %>
+            <strong><%= c.user_full_name %></strong><br/>
+          <% end %>
+          <%= c.comment_text %>
+        </p>
+      </td>
+    </tr>
+    <% end %>
+  </table>
+<% end %>
```

Comment:
Still feeling like it's not worth duplicating the markup for displaying cart details...